.travis.yml

```diff
@@ -1,6 +1,14 @@
 language: python
+sudo: true
```

Comment:
this switches us back to the old infrastructure, but since the tests run super fast anyways i think that's fine.

we need to upgrade to ES 1.x to use travis's built in service